mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -1254,4 +1254,157 @@ Feature: Requests tests
+    * def permanentLoanTypeId = call uuid1
+    * def intStatusName = 'Available'
+    * def itemPrefix = 'itemPref'
+    * def itemSuffix = 'itemSuf'
+
+    * def permanentLoanTypeEntityRequest = read('samples/item/permanent-loan-type-entity-request.json')
+    * permanentLoanTypeEntityRequest.name = permanentLoanTypeEntityRequest.name + ' ' + random_string()
+    Given path 'loan-types'
+    And request permanentLoanTypeEntityRequest
+    When method POST
+    Then status 201
+
+    * def itemEntityRequest1 = read('samples/item/item-entity-request.json')
+    * itemEntityRequest1.barcode = itemBarcode1
+    * itemEntityRequest1.id = itemId1
+    * itemEntityRequest1.holdingsRecordId = holdingsRecordId1
+    * itemEntityRequest1.callNumber = callNumber1
+    * itemEntityRequest1.materialType.id = karate.get('extMaterialTypeId', materialTypeId1)
+    * itemEntityRequest1.status.name = karate.get('extStatusName', intStatusName)
+    * itemEntityRequest1.effectiveCallNumberComponents.callNumber = callNumber1
+    * itemEntityRequest1.effectiveCallNumberComponents.prefix = itemPrefix
+    * itemEntityRequest1.effectiveCallNumberComponents.suffix = itemSuffix
+
+    Given path 'inventory', 'items'
+    And request itemEntityRequest1
+    When method POST
+    Then status 201
+
+    * def itemEntityRequest2 = read('samples/item/item-entity-request.json')
+    * itemEntityRequest2.barcode = itemBarcode2
+    * itemEntityRequest2.id = itemId2
+    * itemEntityRequest2.holdingsRecordId = holdingsRecordId2
+    * itemEntityRequest2.callNumber = callNumber2
+    * itemEntityRequest2.materialType.id = karate.get('extMaterialTypeId', materialTypeId2)
+    * itemEntityRequest2.status.name = karate.get('extStatusName', intStatusName)
+    * itemEntityRequest2.effectiveCallNumberComponents.callNumber = callNumber2
+    * itemEntityRequest2.effectiveCallNumberComponents.prefix = itemPrefix
+    * itemEntityRequest2.effectiveCallNumberComponents.suffix = itemSuffix
+
+    Given path 'inventory', 'items'
+    And request itemEntityRequest2
+    When method POST
+    Then status 201
+
+    # post requests
+    * def requestId1 = call uuid1
+    * def requestId2 = call uuid1
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostRequest') { requestId: #(requestId1), itemId: #(itemId1), requesterId: #(userId), extRequestType: #(requestType), extInstanceId: #(instanceId), extHoldingsRecordId: #(holdingId), extServicePointId: #(servicePointId1) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostRequest') { requestId: #(requestId2), itemId: #(itemId2), requesterId: #(userId), extRequestType: #(requestType), extInstanceId: #(instanceId), extHoldingsRecordId: #(holdingId), extServicePointId: #(servicePointId2) }
+
+    # get requests
+    Given path 'circulation/requests'
+    And param query = 'instance.title =="Long Way to a Small Angry Planet" sortby searchIndex.pickupServicePointName'
```

Comment:
Let's additionally try to sort in descending order to make sure it makes a difference